clinical-domain-agent/src/test/java/care/smith/fts/cda/impl/DeidentifhirStepTest.java

```diff
@@ -43,7 +45,15 @@ void setUp(MockServerClient mockServer) {
     var client = server.createClient(WebClient.builder(), null);
     step =
         new DeidentifhirStep(
-            client, domains, ofDays(14), deidentifhirConfig, scraperConfig, meterRegistry);
+            client,
+            new DeidentifhirStepConfig(
+                new TCAConfig(server, domains),
+                ofDays(14),
+                new File("IDScraper.profile"),
+                new File("CDtoTransport.profile")),
+            deidentifhirConfig,
+            scraperConfig,
```

Comment:
Smells that both, the File and the parsed Config must be passed now.

Reply:
Yes, I was thinking about moving the config parsing into DeidentifhirStep